src/App.vue

```diff
@@ -79,7 +91,7 @@
           <v-row>
             <v-col cols="12" lg="9">
               <header>
-                <h1>Incorporation Application</h1>
+                <h1>{{ getFilingName }}</h1>
```

Comment:
Is other UIs, `name` is actually the filing type.
Here, is it actually the title? Maybe need to clean up some naming.

Reply:
Meh, maybe this is OK. I see that `FilingNames` is an enum with the user-friendly filing name, just as this getter suggests.
The problem lies with the API field `name` that is actually the type. I dunno how to solve this atm. Just beware of this for now.

Reply:
For sure, i just followed the api value in this case. I do agree